web/src/routes/SearchPage.tsx

```diff
@@ -100,16 +100,17 @@ const SearchPage = ({ city, cityCode, languageCode, pathname }: CityRouteProps):
       <Helmet pageTitle={pageTitle} languageChangePaths={languageChangePaths} cityModel={city} />
       {SearchBar}
       <List>
-        {results.map(({ title, content, path, thumbnail }) => (
-          <SearchListItem
-            title={title}
-            contentWithoutHtml={parseHTML(content)}
-            key={path}
-            query={query}
-            path={path}
-            thumbnail={thumbnail}
-          />
-        ))}
+        {query.length > 0 &&
+          results.map(({ title, content, path, thumbnail }) => (
+            <SearchListItem
+              title={title}
+              contentWithoutHtml={parseHTML(content)}
+              key={path}
+              query={query}
+              path={path}
+              thumbnail={thumbnail}
+            />
+          ))}
       </List>
       <SearchFeedback
```

Comment:
:wrench: I think it would be better to also conditionally render the whole `SearchFeedback` instead of returning an empty `div` if the query is empty.